nuclio_sdk/platform.py

```diff
@@ -20,13 +20,31 @@
 
 
 class Platform(object):
-    def __init__(self, kind, namespace="default", connection_provider=None):
+    def __init__(
+        self, kind, namespace="default", connection_provider=None, control_callback=None
+    ):
         self.kind = kind
         self.namespace = namespace
 
         # connection_provider is used for unit testing
         self._connection_provider = connection_provider or http.client.HTTPConnection
 
+        self.control_callback = control_callback
+
+    async def ensure_explicit_ack(self, event):
+        """
+        Ensures marking the offset on the stream according to the given arguments
+
+        :param event
+        :type event
+        """
+        message = event.get_explicit_ack_message()
+        await self.control_callback(message)
```

Comment:
if control_callback is none, raise an exception with a concise message it was not initialize ?